manager/state/raft/raft.go

```diff
@@ -1009,7 +1009,7 @@ func (n *Node) GetVersion() *api.Version {
 		return nil
 	}
 
-	status := n.Node.Status()
+	status := n.raftNode.Status()
```

Comment:
I actually added `Status` method only because tests need it for some cruel magic :) But yeah, I can reuse it everywhere.